internal/config/node/systemd.go

```diff
@@ -19,26 +17,13 @@ var (
 
 func SystemdHasCollectMode() bool {
 	systemdHasCollectModeOnce.Do(func() {
-		stdout, err := exec.Command("systemctl", "--version").Output()
+		// This will show whether the currently running systemd supports CollectMode
+		_, err := exec.Command("systemctl", "show", "-p", "CollectMode", "systemd").Output()
 		if err != nil {
-			systemdHasCollectModeErr = err
+			systemdHasCollectModeErr = errors.Wrapf(err, "check systemd CollectMode")
```

Comment:
nit: I would exclude "exit code 1" as being an error.

Reply:
do you mean we should not set the error? or I shouldn't include the contents of `err` in the returned error ?

Reply:
Not set the error if we got exit code 1 (which 99% means that the option is not supported).